detection_rules/rule.py

```diff
@@ -395,6 +402,58 @@ def setup_in_note(self, value: bool):
     def skip_validate_note(self) -> bool:
         return os.environ.get('DR_BYPASS_NOTE_VALIDATION_AND_PARSE') is not None
 
+    @cached_property
+    def skip_validate_bbr(self) -> bool:
+        return os.environ.get('DR_BYPASS_BBR_LOOKBACK_VALIDATION') is not None
+
+    def validate_bbr(self, bypass: bool = False):
```

Comment:
Does this have to be passed down?

Reply:
`skip_validate_bbr` is only used inside of `DataValidator` since all of the BBR validation occurs there, so the property does not need to be passed.